api/envoy/config/endpoint/v3/load_report.proto

```diff
@@ -159,6 +160,14 @@ message ClusterStats {
   // in the DropOverload policy.
   repeated DroppedRequests dropped_requests = 5;
 
+  // the request latencies in this cluster. Only HTTP requests are currently
+  // supported. This field is not aggregated over the load_reporting_interval
+  // due to the way Envoy histograms are flushed, so these will only
+  // have data for the configured metrics interval.
+  // The set of quantiles are the stringified values from
+  // HistogramStatisticsImpl::supportedQuantiles()
+  map<string, google.protobuf.DoubleValue> request_latency_percentiles = 7;
```

Comment:
Is this field expected to be populated unconditionally, or is this something that will be configurable?  If the latter, how will it be enabled?  Would it make sense to have something in the LRS response that allows the server to tell the client whether to send this data?

Reply:
I was thinking of populating it unconditionally. Do we expect a perf issue if we report these unconditionally?

If yes, then yeah, we want to add something to the LRS response to tell the client to send this.

Reply:
At least on the gRPC side, there would be a performance impact to collecting these stats.  I would not want this to be enabled unless the server was explicitly wanting this data.

Reply:
Sounds good. We'll leave it disabled by default.